mail/index.js

```diff
@@ -0,0 +1,13 @@
+import { render as reactRender } from '@react-email/render';
+
+export { ActivationEmail } from './emails/activation';
+export { ConfirmationEmail } from './emails/confirmation';
+export { NotificationEmail } from './emails/notification';
+export { RecoveryEmail } from './emails/recovery';
+
+export function render(component) {
```

Comment:
Será que não faz sentido cada template já devolver as duas versões (html e text)?

Também dá para o `reactRender` ser chamado de dentro dos templates. Faz sentido? Não me parece ter necessidade de cada local do código que for usar o template ter que importar também o render.

Reply:
Acho que faz sim!